lib/plugins/blocks.js

```diff
@@ -268,6 +268,17 @@ function inject (bot, { version, storageBuilder, hideErrors }) {
     }
   })
 
+  let chunkBatchStartTime = 0
+  bot._client.on('chunk_batch_start', (packet) => {
+    chunkBatchStartTime = Date.now()
+  })
+
+  bot._client.on('chunk_batch_finished', (packet) => {
```

Comment:
can you add a comment explaining what is the purpose of this system ?

Reply:
Added a comment and comments each part of the calculation now that I am doing it correctly.